doc,opensearch: add troubleshooting for opensearch-dashboards (#2124)
aiven/devportal: docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst

Troubleshoot OpenSearch® Dashboards
==================================

OpenSearch® Dashboard is unavailable
---------------------------------------

OpenSearch Dashboards requires that the version of OpenSearch on your cluster be the same. If OpenSearch Dashboards is unavailable, you will see the following error message in the OpenSearch Dashboards logs:

> .. code:: text

Author: is this what you mean by using a code style. When i try to preview it on github. it does not look like it is getting formatted. 

Also how do ignore these vale errors? as this is what the log message looks like.

Reviewer: @RommelLayco 
I included it in a code-block. Your approach is also accurate; when formatting after "..code::", ensure to create a line break and indent the text.
I noticed that the Table of Contents (TOC) entry was missing for this topic. I've included the TOC entry and updated the code formatting accordingly.

Troubleshoot opensearch dashboards
==================================

> OpenSearch® Dashboards is unavailable
> ---------------------------------------

Reviewer: ```suggestion
OpenSearch® Dashboards are unavailable
-------------------------------------------
```

OpenSearch® Dashboards requires the version of OpenSearch® on your cluster to be the same version. If OpenSearch® Dashboards is unavailable

> You may see the following logs in your opensearch-dashboards

Reviewer: OpenSearch Dashboards requires that the version of OpenSearch on your cluster be the same. If OpenSearch Dashboards is unavailable, you will see the following error message in the opensearch-dashboards logs:

> ```

Reviewer: I would suggest adding this in code style, with a line space above.

This version of OpenSearch Dashboards (v1.3.2) is incompatible with the following OpenSearch nodes in your cluster: v1.2.4 @ opensearch-searchdex-1.aiven.local/[${ipaddress}]:9200 (${ip_address}
```

> This situation typically occurs when there is a new version of opensearch released.

Reviewer: ```suggestion
This situation typically occurs when a new version of OpenSearch is released.
```

> When a new version of opensearch is available, any new nodes that are created for the service will use the new version of opensearch. The aiven platform will detect that the nodes in the service have different version and will begin upgrading the old nodes to a new version. Once this upgrade has been completed opensearch-dashboards should become available.

Reviewer: ```suggestion
When a new version of OpenSearch is available, any newly created nodes for the service will automatically use the latest version. The Aiven platform will identify any variance in versions among the service nodes and initiate the process of upgrading outdated ones to the new version. Following the successful completion of this upgrade, Opensearch-dashboards will be accessible.
```
